server/src/lib/jobs/resolve.cpp

```diff
@@ -436,11 +443,24 @@ static std::string groupSql(tnt::Connection& conn, const Group::Rules& group)
             id_asset_element as id
         FROM t_bios_asset_element
         WHERE id_asset_element IN ({})
-    )"_format(fty::implode(subQueries, ") " + sqlLogicalOperator(group.groupOp) + " id_asset_element IN ("));
+    )"_format(
+        fty::implode(subQueries, ") " + sqlLogicalOperator(group.groupOp) + " id_asset_element " + strGroup + "IN ("));
```

Comment:
Is not correct... 
if you have such case
```
conditions = [
{ field = Name, op = Is, value = srv },
{ field = Location, op = Is, value = datacenter },
{ field = Group, op = IsNot, value = 42 },
]
```
Then result query would be ```id in (select id where Name = srv) and not in (select id where location = datacenter) and not in (select if from group = 42```
So, you cannot to join subqueries in same negative condition from 1 subquery